Added some data to sanity for the about section.

ronaldpaek opened the pull request.

TidbitsJS commented on src/pages/About.js:
```diff
@@ -9,15 +9,14 @@ import { style } from '../styles';
 
 		client.fetch(query).then(data => {
-			console.log('hello');
-			setAbouts(data);
+			setSkills(data);
```
Move the sanity fetching functions to a separate file. Import and call them accordingly.

**Always, always catch the errors while doing any fetch logic**. Using the try-catch block with async-await will be better in my eyes. Not mandatory, though.

In the future, we will catch and display these errors on UI. For now, simply console log it.

ronaldpaek commented:
Updated Theme object to work with dark mode, implemented color schemes, reworked using after effect, reworked logic to toggle zIndex so H2 shows under menuBar, setup first 2 pages to toggle with color scheme, refactor all code to be consistent with MUI style like you told me. I just need to break down each component then it's manageable, when it's too long of code I get lost so I adjust css on the component with sx prop and utilize the theme object / function to be able to use color scheme add new colors tested it out did style overrides removed css file, also I figured out how to use flex: 1 and tested out mui grid system, I just don't like it, although I made, but I actually really prefer just using grid now, I will do the rest of the pages.

ronaldpaek commented:
Refactored my context to custom hooks useColorModeContext and useOpenModeContext, since I made context and broke down my components I realize I don't need to pass any functions down as props anymore, refactor to just useContext whenever the component in the component itself. Refactor my ColorModeContext Provider so I can more easily modify the theme object if needed, made separate file and created a theme and imported it to the provider.
Move components into Folders to group subcomponents together for the specific page. Refactor to break down code into more components so it's easier and less stressful to read code, decided to use display flex or grid only now, so no more mui grid, realized it doesn't work the way I like and also display: grid is a lot easier to implement, but played around with flex: 1 etc refactor to make my code look consistent code style and using good semantics along with using clean mui techniques.

TidbitsJS commented on src/components/About/Clients.js:
```diff
@@ -0,0 +1,59 @@
+		<Box>
+			<Box
+				sx={{
+					bgcolor: theme =>
+						theme.palette.mode === 'light' ? '#F8F8F8' : '#0D0D0D',
```
Can we move these colors to the custom color palette you have created? 

```javascript
bgcolor: theme  => theme.custom.bg.primary
```

Something like that you have done it for `color: theme => theme.custom.neutral.primary` field.

TidbitsJS commented on src/components/Header/ThemeModeToggleButton.js:
```diff
@@ -0,0 +1,45 @@
+				height: 44,
+				ml: 2,
+				bgcolor: theme.custom.toggle.bgcolor,
+				color: theme.palette.mode === 'light' ? 'black' : 'white',
```
Use your custom theme

TidbitsJS commented on src/components/Resume/ColumnItem.js:
```diff
@@ -0,0 +1,51 @@
+				component='span'
+				sx={{
+					fontSize: '.875rem',
+					color: theme =>
+						theme.palette.mode === 'light' ? '#44566C' : '#B7B7B7'
```
Use your custom theme

TidbitsJS commented on src/components/Resume/ColumnItem.js:
```diff
@@ -0,0 +1,51 @@
+			</Typography>
+			<Typography
+				sx={{
+					color: theme => (theme.palette.mode === 'light' ? 'black' : '#B7B7B7')
```
Use the custom theme

TidbitsJS commented on src/components/Resume/Knowledges.js:
```diff
@@ -0,0 +1,49 @@
+						sx={{
+							borderRadius: '.5rem',
+							color: 'secondary.main',
+							bgcolor: theme =>
+								theme.palette.mode === 'light' ? '#EDF2F2' : '#1C1C1C',
```
Same. Move those colors to the custom theme and use it here as you used on the rest places

TidbitsJS commented on src/components/Resume/WorkingSkills.js:
```diff
@@ -0,0 +1,57 @@
+							component='span'
+              sx={{
+                fontWeight: 600,
+								color: theme =>
+									theme.palette.mode === 'light' ? '#526377' : '#A6A6A6'
```
Use custom theme

TidbitsJS commented on src/components/Resume/WorkingSkills.js:
```diff
@@ -0,0 +1,57 @@
+							component='span'
+              sx={{
+                fontWeight: 600,
+								color: theme =>
+									theme.palette.mode === 'light' ? '#526377' : '#A6A6A6'
```
Use custom theme
